tensornetwork/backends/tensorflow/tensorflow_backend_test.py

```diff
@@ -113,3 +115,106 @@ def test_einsum():
   actual = backend.einsum('ij,jil->l', a, b)
   expected = np.array([4.0, 4.0])
   np.testing.assert_allclose(expected, actual)
+
+
+def test_norm():
+  backend = tensorflow_backend.TensorFlowBackend()
+  a = backend.convert_to_tensor(np.ones((2, 2)))
+  assert backend.norm(a).numpy() == 2
+
+
+def test_eye():
+  backend = tensorflow_backend.TensorFlowBackend()
+  a = backend.eye(N=4, M=5)
+  np.testing.assert_allclose(np.eye(4, 5), a)
+
+
+def test_ones():
+  backend = tensorflow_backend.TensorFlowBackend()
+  a = backend.ones((4, 4))
+  np.testing.assert_allclose(np.ones((4, 4)), a)
+
+
+def test_zeros():
+  backend = tensorflow_backend.TensorFlowBackend()
+  a = backend.zeros((4, 4))
+  np.testing.assert_allclose(np.zeros((4, 4)), a)
+
+
+def test_randn():
+  backend = tensorflow_backend.TensorFlowBackend()
```

Comment:
A lot of this code is very redudent. Can you please parameterize these tests to reduce code copying.